op-program/prestates/standard.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Little bit bike shedding, but would it be reasonable to just have a single JSON file with all tags for whether it's standard or not?

Also now I think of it - what exactly is the definition of standard? I assume only governance approved so maybe we should call it that.

Reply:
We could build a combined file with https://github.com/ethereum-optimism/optimism/pull/12538 though it can't know what is or isn't considered standard. I can then update the reproducibility check job to just check the JSON matches what's checked in (possibly with a little filtering if we need to manually edit - see below).

To track what is or isn't standard we either need to manually add a field to reflect that status in the committed version or make the rule that superchain-registry only accepts versions with no `-` in them (e.g `-rc`, `-alpha` etc) and not starting with `0`.